app/src/main/java/fr/free/nrw/commons/depictions/models/Search.java

```diff
@@ -0,0 +1,140 @@
+package fr.free.nrw.commons.depictions.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Model class for object obtained while parsing depiction response
+ * this class contains all the details of for the media object
+ */
+
+public class Search {
```

Comment:
I assume that this is just the `MwQueryPage` class

Reply:
I see no `wordcount` or `snippet` in `MwQueryPage` so I'll guess "no"